### PR TITLE
Two functions for drawing lines

### LINKED ISSUE #1114: Is there a way to draw a line?
Is there a built-in / plugin method to draw lines?

### PR DESCRIPTION
**Pull request by majdhajyehia**
### What's new?
Added two different methods for drawing multiple kinds of lines, both two methods have different algorithms for the optimization of lines in order to make them not look "so pixelated", users can choose which algorithm to use, but the wuDrawLine is more recommended because I didn't check all cases in the drawLine methods.

### What's next?
- Adding more methods for drawing triangles (you can draw triangles now by drawing multiple lines, but I will be making a separate method for triangles)
- Adding methods for drawing rectangles
- Adding more complicated geometric shapes.

Closes #1114

### CODE REVIEW

**Comment by BestDr** [comment]
This pull request is related to #699

File: packages/plugin-line/src/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,592 @@
+{
```
Comment (hipstersmoothie) [inline packages/plugin-line/src/package-lock.json:1]: it shouldn't have this

File: packages/plugin-line/src/index.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+// Jimp.read('blank.jpeg', (error, img) => {
+// 	if(error) throw error;
+// 	console.log(img);
+// 	//wuDrawLine(50, 500, 1000, 500, img);
+// 	wuDrawLine(1000, 200, 1000, 540, img);
+// 	//drawLine(300, 200, 1080, 540, img);
+// 	img.write("result.jpeg");
+// 	console.log('done');
+// });
```
Comment (hipstersmoothie) [inline packages/plugin-line/src/index.js:117]: remove this

File: packages/plugin-line/src/package.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (hipstersmoothie) [inline packages/plugin-line/src/package.json:1]: Match the style in other plugins

File: packages/plugin-line/src/test.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import DrawLine from 'index';
+import jimp =
```
Comment (hipstersmoothie) [inline packages/plugin-line/src/test.js:2]: Add real test

**Review by hipstersmoothie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**